features/google/src/model/importers/access-log-importer.js

```diff
@@ -0,0 +1,45 @@
+import AccessLogEntry from "../entities/access-log-entry";
+
+const accessLogRegex = /\/Access Log Activity\/.*\.csv$/;
```

Comment:
I'm checking out my own takeout, and unlike other, which seems to not translate this part, this reads "Actividad de registro de accessos". It's also divided in two zip files, one of which is simply HTML.

Reply:
is one an extension of the other or is it a duplication?

Reply:
No, different content. The csv is only in one of them, though. I think the first.

Reply:
ok then I guess there will be two importers for this like the activity importer

Reply:
so is it accessLog information or is it other details? Do you have a screenshot to share?

Reply:
I think it's simply a web for visualization. Does not seem to have any additional information.

Reply:
oh ok then if it does not have any useful information, we can leave it out of the importer right?

Reply:
Looking at this again we do not need a regex in this case and could proceed a with the Facebook importers where we use directly the path to access the file (not sure 100% if something was changed more there but we should be able to access the files by their path). For example for german we check if we have the path `Zugriffsprotokollaktivitäten/Aktivitäten – eine Liste der Google-Dienste, auf d.csv`. We only need this CSV file in the access log importer now. We could look at this also later on as we add more importers

Reply:
Still we would need to know the language in advance. Is there not any Google takeout that contains that kind of metadata?

Reply:
Nope, no such metadata that we know about at the moment. This would be more related to https://github.com/polypoly-eu/polyPod/pull/908; in that case the importer would not have a `regex`, but something like a `path` that it would check.

Reply:
Hum, I'm checking the archive_browser.html that we were dismissing before and it does contain everything there, name of files and everything. It kind of does so in specific divs, so we can maybe extract it for there? (Maybe later)

Reply:
Yes we could use the title (`archive-browser.html` or `Archiv_Übersicht.html`) to detect the language. The annoying thing is what happens if Google splits the data in multiple archives, and places the main html file in an archive that the user does not import.

When exporting the takeout will be in the language set in the Google profile. For example I switched my profile language to German and Romanian and then the export was in that language (For my browser the language is English).

Reply:
Well, it's archive_browser.html even if the content is in Spanish. *And* the google profile, even if you download it, does not have information on the language.
However, good thing about that file is that we wouldn't need to use the language to find the files: file paths are right there on the HTML source.
Also, we can now use multiple zip files at once, thanks to the latest additions by @timoteipalade. So scraping that file, in whatever language it is, might give us surely a lot of information.

Reply:
It looks like this:
![Captura de pantalla de 2022-05-23 12-21-45](https://user-images.githubusercontent.com/500/169799134-2e872a37-feba-44a8-93e7-03469d85a3c7.png)
Even if the text changes, the icon, which uses an inline image, is the same, so we can even get the type of service we're downloading in a roundabout way.

Reply:
Ah, and we have `data-english-name` for an unified name of the service we're looking for:
![Captura de pantalla de 2022-05-23 12-23-22](https://user-images.githubusercontent.com/500/169799453-906dc788-e6e3-4dcc-bd2e-ced82982e2bb.png)

Reply:
That `english-name` is the one used for the divs that contain the names of the files:
![Captura de pantalla de 2022-05-23 12-27-06](https://user-images.githubusercontent.com/500/169800045-fde1cf98-ede0-4dbd-be4c-a31df8d864d1.png)